llvm/unittests/Transforms/Utils/SSAUpdaterBulkTest.cpp

```diff
@@ -236,7 +236,7 @@ TEST(SSAUpdaterBulk, SingleBBLoop) {
   Loop->dump();
```

Comment:
Can you remove the new headers you added, I think fatal error cases now

Reply:
I've removed gtest-spi.h used for EXPECT_NONFATAL_FAILURE, but SourceMgr.h is used for 

```
SMDiagnostic Err;
std::unique_ptr<llvm::Module> M = llvm::parseAssemblyString(IR, Err, Context);
```